src/awkward/operations/ak_from_rdataframe.py

```diff
@@ -1,18 +1,37 @@
 # BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
 
 import awkward as ak
+from awkward._nplikes.numpylike import NumpyMetadata
 
+np = NumpyMetadata.instance()
 
-def from_rdataframe(rdf, columns, offsets_type="int64_t", keep_order=False):
+
+def from_rdataframe(
+    rdf,
+    columns,
+    *,
+    keep_order=False,
+    offsets_type="int64",
+    highlevel=True,
+    behavior=None,
+    with_name=None,
+):
     """
     Args:
         rdf (`ROOT.RDataFrame`): ROOT RDataFrame to convert into an
             Awkward Array.
         columns (str or iterable of str): A column or multiple columns to be
             converted to Awkward Array.
-        offsets_type (str): A C++ type of the ListOffsetArray offsets.
         keep_order (bool): If set to `True` the columns with Awkward type will
             keep order after filtering.
+        offsets_type (str): A `NumpyType.primitive` type of the ListOffsetArray
+            offsets: `"int32"`, `"uint32"` or `"int64"`.
+        highlevel (bool): If True, return an #ak.Array; otherwise, return
+            a low-level #ak.contents.Content subclass.
+        behavior (None or dict): Custom #ak.behavior for the output array, if
+            high-level.
+        with_name (None or str): Gives tuples and records a name that can be
+            used to override their behavior (see #ak.Array).
```

Comment:
```suggestion
        offsets_type (str): A `NumpyType.primitive` type of the ListOffsetArray
            offsets: `"int32"`, `"uint32"` or `"int64"`.
        with_name (None or str): Gives tuples and records a name that can be
            used to override their behavior (see #ak.Array).
        highlevel (bool): If True, return an #ak.Array; otherwise, return
            a low-level #ak.contents.Content subclass.
        behavior (None or dict): Custom #ak.behavior for the output array, if
            high-level.
```